client/components/feature-example/README.md

```diff
@@ -0,0 +1,23 @@
+Feature Example
+=======
+
+Feature Example is a component used to render an mocked example of any feature. It renders whatever childrens it receives, covered but a layer of gradiented opacity.
```

Comment:
This sentence could be improved a bit. 
Feature Example is a component used to render an mocked example of any feature. It renders whatever children it receives. The example is covered by a layer of fading gradient that gives the user a sense of UI that they are missing.